test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e -x
+
+export RAILS_ENV=test
+export BUNDLE_WITHOUT=development
+
+bundle install
+bundle exec rubocop .
```

Comment:
Not needed but just a preference to be explicitly rather than implicitly. Is sounds good to you?